kitten_options_utils.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Sequence, Tuple, Iterable
+
+from kitty.conf.utils import KittensKeyDefinition, key_func, parse_kittens_key
+from kitty.rgb import Color, parse_sharp
+
+func_with_args, args_funcs = key_func()
+FuncArgsType = Tuple[str, Sequence[Any]]
+
+
+def color_parser(val: str) -> Color:
```

Comment:
Do not forget to remove once color options use `to_color`.

Reply:
Done in ea2b421466662e5706c6134d6b72808f45450cd3